cmd/jobserver/main.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/mattermost/mattermost-mattermod/metrics"
+	"github.com/mattermost/mattermost-mattermod/server"
+	"github.com/mattermost/mattermost-server/v5/mlog"
+	"github.com/robfig/cron/v3"
+	"golang.org/x/net/context"
+)
+
+func main() {
+	var configFile string
+	flag.StringVar(&configFile, "config", "config-mattermod.json", "")
+	flag.Parse()
+
+	config, err := server.GetConfig(configFile)
+	if err != nil {
+		mlog.Error("Unable to load Job Server config", mlog.Err(err), mlog.String("file", configFile))
+		os.Exit(1)
+	}
+	if err = server.SetupLogging(config); err != nil {
+		mlog.Error("Unable to configure logging", mlog.Err(err))
+		os.Exit(1)
+	}
+
+	// Metrics system
+	metricsProvider := metrics.NewPrometheusProvider()
```

Comment:
To get this metrics we have to include this service as target in Prometheus // @khos2ow

Reply:
Oh right, good catch. Let me update the Dockerfile.

Reply:
That's true, and will be part of the deployment of this new jobserver.